src/cache/mod.rs

```diff
@@ -488,7 +513,7 @@ mod tests {
         let mut cache = get_test_setup();
         add_cache_entries(&mut cache);
         assert_eq!(cache.cache.len(), 2);
-        for (pm, pmc) in &cache.cache {
+        for (_pm, pmc) in &cache.cache {
```

Comment:
Does `pmc` stand for `peer_message_cache`? Future readers will thank you for clearer variable names.